.gitignore

```diff
@@ -80,6 +80,7 @@ selenium.egg-info/
 third_party/java/jetty/jetty-repacked.jar
 *.user
 *.cache
+bin/
```

Comment:
Since this is a different issue from fixing build errors, I think you should remove the change from this PR and make a new PR with what you think is correct along with an explanation why. Then we can maintain documentation for each of these things.

Reply:
Fixed, now it looks safer.